SoftLayer/managers/ticket.py

```diff
@@ -22,24 +22,30 @@ def __init__(self, client):
         self.account = self.client['Account']
         self.ticket = self.client['Ticket']
 
-    def list_tickets(self, open_status=True, closed_status=True):
+    def list_tickets(self, open_status=True, closed_status=True, limit=None, all_tickets=None):
```

Comment:
Set the default for `all_tickets` to just be False, so you don't have to use the `if` statement later in this function.
```suggestion
    def list_tickets(self, open_status=True, closed_status=True, limit=None, all_tickets=False):
```